src/resources/locales/en.json

```diff
@@ -151,6 +151,10 @@
             "bookmarkTitle": "bookmark",
             "detachWindowTitle": "detach the window",
             "fullscreenTitle": "open fullscreen mode",
+            "goTo": "Go to",
+            "goToError": "This page does not exist",
+            "goToPlaceHolder": "Go to page",
```

Comment:
I think that the text input field "placeholder" should be something like "enter a page number", not a description of the associated action.

Reply:
Fixed via https://github.com/readium/readium-desktop/commit/d38073a801fbf57a96525287a026e9b0da06f196